swagger/forward/swagger.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
```

Comment:
I'd remove the server to relay instead of forward. "Forwarding" means a conscious action to me, while "relaying" is rather passive undertaken by the layer beneath.